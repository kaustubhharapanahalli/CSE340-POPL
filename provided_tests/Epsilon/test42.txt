tsTok3 ((b).(c))* #
" bc "