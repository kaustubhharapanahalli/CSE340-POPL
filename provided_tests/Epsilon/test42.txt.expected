EPSILON IS NOOOOOOOT A TOKEN !!! tsTok3
